### DISCUSSION TITLE
difference between private vs. buddy share.

### DISCUSSION THREAD

**Discussion by critkitten** [nicotine-plus/nicotine-plus#2550] [Q&A]
I have some question about nicotine.

I see a lot of users who make files [Private]. Is this feature supported by nicotine?
I only found Buddy only share which is different to make files Private right?

Buddy-share = files only visible and downloadable to buddy's (plus trusted buddy's)
Private = files visible to all users but only downloadable to user on buddy list.

**Comment by slook** [comment]
Sorry about the delay in responding to your query. See relevant discussions in issues #1870 and #1595.

> make files [Private]. Is this feature supported by nicotine?

No.

> I only found Buddy only share which is different to make files Private right?

Yes, your understanding is correct.
